extensions/Khronos/KHR_materials_cmnBlinnPhong/README.md

```diff
@@ -17,9 +17,9 @@ Written against the glTF 2.0 spec.
 
 This extension defines a Blinn-Phong model based material for use with glTF 2.0 in the same way as the core specification defines Physically Based Rendering (PBR) based materials. Other extension may define additional commonly used materials. For example, if a need for a Phong model arises, an extension named e.g. `cmnPhong` could be specified. 
 
-Given the long history of the Blinn-Phong model and the vast amount of collective experience with applying this model, the demand for convenient use of such materials is still there.
+Given the long history of the Blinn-Phong model and the vast amount of collective experience with applying this model, the demand for convenient use of such materials is still exists.
 
-This extension defines a single material type, Blinn-Phong. Other or simpler materials types such as Lambert or Constant can be described by having zero factors.
+This extension defines a single material type, Blinn-Phong. Other or simpler materials types such as Lambert or Constant may be emulated by applying zero factors. Please note, strict Lambert or Constant materials may be specified in dedicated extensions. 

```

Comment:
I think I would just omit the second two sentences, so that this paragraph becomes:

> Given the long history of the Blinn-Phong model and the vast amount of collective experience with applying this model, the demand for convenient use of such materials is still exists. This extension defines a single material type, Blinn-Phong.

(Mentioning that simpler materials can be emulated via Blinn-Phong can be mentioned in the extensions for those materials)